erpnext/patches/v11_0/make_italian_localization_fields.py

```diff
@@ -5,10 +5,37 @@
 from erpnext.regional.italy.setup import make_custom_fields, setup_report
 import frappe
 
+state_codes = {'Siracusa': 'SR', 'Bologna': 'BO', 'Grosseto': 'GR', 'Caserta': 'CE', 'Alessandria': 'AL', 'Ancona': 'AN', 'Pavia': 'PV',
```

Comment:
I think this belongs to the regional module instead. Does it work for fresh installs?

Reply:
See https://github.com/frappe/erpnext/blob/develop/erpnext/regional/india/gst_state_code_data.json for example.